pulpcore/pulpcore/app/viewsets/custom_filters.py

```diff
@@ -60,3 +63,101 @@ class IsoDateTimeFilter(DateTimeFilter):
     def __init__(self, *args, **kwargs):
         kwargs.setdefault('help_text', _('ISO 8601 formatted dates are supported'))
         super().__init__(*args, **kwargs)
+
+
+class RepoVersionHrefFilter(Filter):
```

Comment:
Should I rename this to "BaseRepoVersionHrefFilter" since it's meant to be subclassed and not used directly?